### PR TITLE
Submit Pepys version 1.3.1

### PR DESCRIPTION
**Pull request by LukeBriggsDev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41980)

### CODE REVIEW

File: manifests/l/LukeBriggs/Pepys/1.3.1/LukeBriggs.Pepys.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+PackageIdentifier: LukeBriggs.Pepys
+PackageVersion: 1.3.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/LukeBriggsDev/Pepys/releases/download/v1.3.1/pepys.1.3.1.win.x86_64.installer.exe
+  InstallerSha256: 1095160248B3AADF7A6BC07E7496856F4FE8DBF6B80943938EAC0C3AF04B4ED4
+ManifestType: installer
+ManifestVersion: 1.0.1
```
Comment (Trenly) [inline manifests/l/LukeBriggs/Pepys/1.3.1/LukeBriggs.Pepys.installer.yaml:12]:
```suggestion
ManifestVersion: 1.0.0
```

File: manifests/l/LukeBriggs/Pepys/1.3.1/LukeBriggs.Pepys.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+PackageName: Pepys
+PackageUrl: https://www.lukebriggs.dev/pepys/
+License: GPLv3
+LicenseUrl: https://www.gnu.org/licenses/gpl-3.0.en.html
+Copyright: Luke Briggs 2021
+ShortDescription: A straightforward markdown journal
+Description: Pepys is a journaling application that utilises markdown for writing and storing the journal entries
+Moniker: pepys
+ManifestType: defaultLocale
+ManifestVersion: 1.0.1
```
Comment (Trenly) [inline manifests/l/LukeBriggs/Pepys/1.3.1/LukeBriggs.Pepys.locale.en-US.yaml:20]:
```suggestion
ManifestVersion: 1.0.0
```

File: manifests/l/LukeBriggs/Pepys/1.3.1/LukeBriggs.Pepys.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: LukeBriggs.Pepys
+PackageVersion: 1.3.1
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.1
```
Comment (Trenly) [inline manifests/l/LukeBriggs/Pepys/1.3.1/LukeBriggs.Pepys.yaml:8]:
```suggestion
ManifestVersion: 1.0.0
```

File: manifests/l/LukeBriggs/Pepys/1.3.1/LukeBriggs.Pepys.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: LukeBriggs.Pepys
+PackageVersion: 1.3.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/LukeBriggsDev/Pepys/releases/download/v1.3.1/pepys.1.3.1.win.x86_64.installer.exe
```
Comment (Trenly) [inline manifests/l/LukeBriggs/Pepys/1.3.1/LukeBriggs.Pepys.installer.yaml:9]: This URL is no longer valid, it looks like the release has been removed

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**